packages/blade/src/utils/getMediaQuery/getMediaQuery.ts

```diff
@@ -0,0 +1,3 @@
+export const getMediaQuery = ({ min, max }: { min: number; max?: number }): string => {
+  return `screen and (min-width: ${min}px)${max ? ` and (max-width: ${max}px)` : ''}`;
```

Comment:
Could you please also add a few unit test cases for this util?

Reply:
We're not including `@media`, intentional?

Reply:
yup. we do `window.matchMedia` on this in useBreakpoints hook which expects string like this without `@media`

Will add tests

Reply:
done